src/components/icon/icon.component.tsx

```diff
@@ -55,6 +55,8 @@ export interface IconProps extends Omit<StyledIconProps, "type">, MarginProps {
   inputSize?: "small" | "medium" | "large";
   /** @ignore @private */
   tabIndex?: number;
+  /** @ignore @private internal prop to be used in the event no aria-label should be specified */
+  hideAriaLabel?: boolean;
```

Comment:
Yeah I had considered tearing out the aria-label out of `Icon` all together but this would be a breaking change as aria-label is advertised as a prop. I figured for now being able to hide the aria-label where we needed to would suffice. I had checked that if I removed the aria-label from `Icon` would it also address the `Pod` issue too but it didn't. Hence I went for this approach instead.

Reply:
Changed this approach to having `aria-label={undefined}` now passed from `Link`. I've removed this prop from `Icon` now based on the other suggested changes.